spec/lib/granite/action/subject_spec.rb

```diff
@@ -32,4 +32,18 @@
     specify { expect(Action.new(comment: 'Comment', subject: student).comment).to eq('Comment') }
     specify { expect(Action.new(student, comment: 'Comment').comment).to eq('Comment') }
   end
+
+  describe '::subject?' do
+    specify { expect(Action).to be_subject }
+
+    context 'when Action does not define subject' do
+      before do
+        stub_class(:action, Granite::Action) do
+          attribute :comment, String
+        end
```

Comment:
This is adding extra information to the spec but is not required for it. I think the best way to handle this would be:
1. Move `stub_class` in `describe Granite::Action::Subject` to `describe '#initialize'`
2. Have a very simple `stub_class` inside `describe '.subject?'` that only creates a class.
3. Change context to "when action defined subject", and have something like `before { Action.subject :student }`

Reply:
:+1: